lib/govuk_tech_docs/tech_docs_html_renderer.rb

```diff
@@ -95,9 +95,9 @@ def block_code(text, lang)
         # be `defined?`, so we can jump straight to rendering HTML ourselves.
 
         fragment = Nokogiri::HTML::DocumentFragment.parse("")
-        pre = Nokogiri::XML::Node.new "pre", fragment
+        pre = Nokogiri::XML::Node.new "pre", fragment.document
         pre["tabindex"] = "0"
-        code = Nokogiri::XML::Node.new "code", fragment
+        code = Nokogiri::XML::Node.new "code", fragment.document
```

Comment:
Wasn't sure how to test these, it appears to be a fallback for when you don't have a source code highlighter, but I think it should be okay given the other change worked.